src/main/java/com/zipcodewilmington/StringArrayUtils.java

```diff
@@ -42,6 +50,13 @@ public static String getSecondToLastElement(String[] array) {
      * @return true if the array contains the specified `value`
      */ // TODO
     public static boolean contains(String[] array, String value) {
+
```

Comment:
Complex is better than complicated...

`return getNumberOfOccurrences(array, value) > 0`